src/index.js

```diff
@@ -61,6 +62,15 @@ const singleUserHandler = async (
     ],
   };
 
+  // Track message type send by user
+  if (type === 'message') {
+    ga(userId, {
+      ec: 'UserInput',
+      ea: 'MessageType',
+      el: otherFields.message.type,
+    });
+  }
```

Comment:
This will send "UserInput" event every time. For example, if the user is using PC version of LINE, for the interaction flows like:

1. User forwards a rumor
2. User selects article search result 1
3. User selects reply 1
4. User types `y` to say the reply is useful

They all sends `UserInput` `MessageType` event, with other events like `Article`, `Reply` events in between.

If we want a clean flow in [events flow visualization](https://www.youtube.com/watch?v=45QYuTmmd6Y) showing `UserInput/ArticleSearch` -> `Article/Selected` -> `Reply/Selected` -> `UserInput/FeedbackVote`, we should not record `UserInput/MessageType` here.